app/src/main/java/org/simple/clinic/facility/change/FacilityChangeScreen.kt

```diff
@@ -140,11 +140,9 @@ class FacilityChangeScreen(context: Context, attrs: AttributeSet) : RelativeLayo
 
   fun showProgressIndicator() {
     progressView.visibility = VISIBLE
-    facilityRecyclerView.visibility = GONE
```

Comment:
The `RecyclerView` will now only get populated once the progress indicator is hidden from the controller so this is no longer needed.